projects/mvc/result-json/result-json.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <AssemblyName>result</AssemblyName>
```

Comment:
Remove these properties. They aren't needed,

Reply:
Remove everything except TargetFramework

Reply:
Removed the unnecessary properties.